src/ioctls/vcpu.rs

```diff
@@ -2172,4 +2227,57 @@ mod tests {
             vcpu.enable_cap(&cap).unwrap();
         }
     }
+    #[cfg(target_arch = "x86_64")]
+    #[test]
+    fn test_get_tsc_khz() {
+        use std::os::unix::io::FromRawFd;
+
+        let faulty_vcpu_fd = VcpuFd {
+            vcpu: unsafe { File::from_raw_fd(-1) },
+            kvm_run_ptr: KvmRunWrapper {
+                kvm_run_ptr: mmap_anonymous(10),
+                mmap_size: 10,
+            },
+        };
+        assert!(faulty_vcpu_fd.get_tsc_khz().is_err());
```

Comment:
Can you just add this call to the already existing test function so that we do not need to create another faulty_vcpu_fd? Sorry, I wasn't being explicit in my previous comment.

Reply:
Yes, I moved it to the other test.